x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/test_connector_form.test.tsx

```diff
@@ -88,6 +116,45 @@ describe('test_connector_form', () => {
     expect(result?.exists()).toBeTruthy();
   });
 
+  it('renders the execution test field', async () => {
+    const actionTypeExecutionMode = {
+      id: 'execution-mode-type',
+      iconClass: 'test',
+      selectMessage: 'test',
+      validateParams: (): Promise<GenericValidationResult<unknown>> => {
+        const validationResult = { errors: {} };
+        return Promise.resolve(validationResult);
+      },
+      actionConnectorFields: null,
+      actionParamsFields: mockedActionParamsFieldsExecutionMode,
+    };
+    const actionTypeRegistryExecutionMode = actionTypeRegistryMock.create();
+    actionTypeRegistryExecutionMode.get.mockReturnValue(actionTypeExecutionMode);
+
+    const connector = {
+      actionTypeId: actionTypeExecutionMode.id,
+      config: {},
+      secrets: {},
+    } as ActionConnector;
+
+    render(
+      <I18nProvider>
+        <TestConnectorForm
+          connector={connector}
+          executeEnabled={true}
```

Comment:
Should we test the case where `executeEnabled=false` or `undefined`?

Reply:
I think I might be missing what you're getting at here. The `executeEnabled` doesn't have any affect on the field I added (`executionMode`) right?

Reply:
Yes, sorry wrong line of code. I mean the `executionMode`.

Reply:
Got it, yep I'll add some tests for those scenarios 👍

Reply:
You are right! Ensuring that the other text field shouldn't be displayed is enough.